saw-core/src/Verifier/SAW/Grammar.y

```diff
@@ -356,9 +357,9 @@ parseTupleSelector t i =
 
 -- | Create a module name given a list of strings with the top-most
 -- module name given first.
-mkPosModuleName :: [PosPair String] -> PosPair ModuleName
+mkPosModuleName :: [PosPair Text] -> PosPair ModuleName
 mkPosModuleName [] = error "internal: Unexpected empty module name"
-mkPosModuleName l = PosPair p (mkModuleName nms)
+mkPosModuleName l = PosPair p (mkModuleName (fmap Text.unpack nms))
```

Comment:
Should `mkModuleName` take `[Text]` instead?

Reply:
I think I would prefer to make changes to `mkModuleName` as a separate PR; this PR on its own does not break anything in saw-script, but changing the type of `mkModuleName` would be a breaking change. I'll open a separate ticket for that.

Reply:
See #170.